Dockerfile

```diff
@@ -17,37 +17,67 @@ ARG PG_MAJOR=13
 ## in relation to the total image size.
 ## By choosing a very basic base image, we do keep full control over every part
 ## of the build steps. This Dockerfile contains every piece of magic we want.
-FROM rust:1.54-slim-buster AS compiler
+FROM ubuntu:21.04 AS compiler
```

Comment:
This should really be 21.10, however 21.10 is currently utterly broken (it is in alpha, so expected)

Details: 21.10 fails with `apt-get update`:
```
#19 1.094 E: Problem executing scripts APT::Update::Post-Invoke 'rm -f /var/cache/apt/archives/*.deb /var/cache/apt/archives/partial/*.deb /var/cache/apt/*.bin || true'
#19 1.094 E: Sub-process returned an error code
------
error: failed to solve: executor failed running [/bin/sh -c apt-get update]: exit code: 100
make: *** [Makefile:128: prepare] Error 1
```

I'm not in the mood to fix alpha related problems currently, so using 21.04 to get the build passed would be good enough for now?